opensrp-giz-malawi/src/main/java/org/smartregister/giz/repository/GizMalawiRepository.java

```diff
@@ -66,7 +68,7 @@ public void onCreate(SQLiteDatabase database) {
 
         runLegacyUpgrades(database);
 
-        onUpgrade(database, 7, BuildConfig.DATABASE_VERSION);
+        onUpgrade(database, 8, BuildConfig.DATABASE_VERSION);
```

Comment:
If this migration is necessary upgrade the database version on the giz gradle file